ui/components/multichain/account-picker/account-picker.js

```diff
@@ -0,0 +1,49 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import {
+  ButtonBase,
+  ICON_NAMES,
+  AvatarAccount,
+  Text,
+} from '../../component-library';
+import Box from '../../ui/box/box';
+import {
+  BackgroundColor,
+  DISPLAY,
+  FONT_WEIGHT,
+  Size,
+} from '../../../helpers/constants/design-system';
+
+export const AccountPicker = ({ account, onClick }) => {
+  return (
+    <ButtonBase
+      className="multichain-account-picker"
+      onClick={onClick}
+      backgroundColor={BackgroundColor.transparent}
+      endIconName={ICON_NAMES.ARROW_DOWN}
+      ellipsis
+    >
+      <Box display={DISPLAY.FLEX} gap={2}>
```

Comment:
instead of wrapping Text and AvatarAccount in a Box can you just use the `display` and `gap` props directly on ButtonBase and save some html bloat?
```suggestion
      ellipsis
      display={DISPLAY.FLEX}
      gap={2}
    >
```

Reply:
Unfortunately not: 
<img width="359" alt="Breakage" src="https://user-images.githubusercontent.com/46655/227245167-4ec828ad-179f-426a-ad43-54845f24e222.png">